apps/calendar/test/unit/provider/caldav_test.js

```diff
@@ -346,11 +343,10 @@ suite('provider/caldav', function() {
       });
 
       test('offline handling', function(done) {
-        var realOffline = app.offline;
-        app.offline = function() { return true; };
+        subject.isOffline = function() { return true; };
         subject.getAccount(input, function cb(cbError, cbResult) {
           done(function() {
-            app.offline = realOffline;
+            delete subject.isOffline;
```

Comment:
I should really change the test structure to use `setup` and `teardown` instead of this hack